src/plone/restapi/tests/http-examples/translated_messages_types_folder.resp

```diff
@@ -14,6 +14,7 @@ Content-Type: application/json+schema
         },
         {
             "behavior": "plone",
+            "description": "None",
```

Comment:
@JeffersonBledsoe This doesn't look right... If the field has no description, I would expect this to get serialized as `null` or an empty string, not a string containing "None"

You might need to do `translate(group.description, context=getRequest()) if group.description is not None else ""` below

Reply:
Hey @davisagli, thanks for catching this! It didn't show up in my cursory local testing of the PR